### PR TITLE
DIRECTOR: LINGO: Implement kTheModified field in CastMember::getField()

### PR DESCRIPTION
**Pull request by r41k0u**
This is a quick fix to implement the STUB for the `Modified` property of CastMember

### CODE REVIEW

**Comment by sev-** [comment]
Please explain how it was tested.

**Comment by sev-** [comment]
Please explain how it was tested.

**Comment by sev-** [comment]
This got conflicts. You need to rebase.

File: engines/director/castmember.h [hunk 1]
```diff
@@ -73,7 +73,7 @@ class CastMember : public Object<CastMember> {
 	virtual bool isEditable() { return false; }
 	virtual void setEditable(bool editable) {}
 	virtual bool isModified() { return _modified; }
-	virtual void setModified(bool modified) { _modified = modified; }
+	void setModified(bool modified);
```
[the remark this answers is not in the document]
Comment (r41k0u) [reply engines/director/castmember.h:76]: You are right. It wasn't necessary. But it wasn't getting inherited and overwritten so I saw no harm in doing that

**Comment by sev-** [comment]
Thanks

**Pull request merged**